replacing authorization by privacy (#61)
CONP-PCNO/conp-documentation: CONP_DATS_fields.md

|Size|distributions - size|**REQUIRED** Total size of the dataset.| The quantity of space required on disk (or other medium) for this dataset.|
|Units|distributions - unit|**REQUIRED** Unit in which the size is measured.(*example*: GB)|Unit in which the size is measured.(KB -> KiloByte, MB -> MegaByte, GB -> GigaByte, TB -> TeraByte and PB -> PetaByte). |

> |Landing Page|distributions - access - landingPage|**REQUIRED** Web address (URL) where the original dataset can be found. Default value points to the README.md for the dataset. |A URL (Web page) that contains information about the associated dataset (with a link to the dataset) or a direct link to the dataset itself. |

Reviewer: ```suggestion
|Landing Page|distributions - access - landingPage|**REQUIRED** Web address (URL) where the original dataset can be found. Default value points to the README.md for the dataset. |A URL (Web page) that contains information about the associated dataset (with a link to the dataset) or a direct link to the dataset itself. |
|Authorizations|distributions - access - authorizations| **REQUIRED**Types of verification that accessing the resource is allowed. Authorization occurs before successful authentication and refers to the process of obtaining approval to use a data set. Ideally specified from a controlled vocabulary or ontology.|This must be one of "Public", "Registered" or "Private". When this field is absent the value will be treated as "Public". "Public" = available to anyone; "Registered" = available to user authorized by the creator; "Private" = available only to the creator.|
```

> |Authorizations|distributions - access - authorizations| **REQUIRED**Types of verification that accessing the resource is allowed. Authorization occurs before successful authentication and refers to the process of obtaining approval to use a data set. Ideally specified from a controlled vocabulary or ontology.|This must be one of "Public", "Registered" or "Private". When this field is absent the value will be treated as "Public". "Public" = available to anyone; "Registered" = available to user authorized by the creator; "Private" = available only to the creator.|

Reviewer: Authorization should not be removed from the documentation as this is a separate field than privacy that goes under the distribution section of the DATS file while privacy is directly at the root of the DATS file.

|Number of Subjects|extraProperties - subjects|**REQUIRED** The number of subjects in the dataset.|Total number of subjects constituting the dataset.|
|CONP Status|extraProperties - CONP_status|**REQUIRED** CONP Status.|The CONP status is used to add the CONP logo or Canadian flag on the left of the dataset and sorting in the data search. Valid values are "CONP" = created using funding from the CONP; "Canadian" = created in Canada without CONP funding; "external" = created outside of Canada.|

> |Privacy|privacy|**REQUIRED** Types of verification that accessing the resource is allowed. Authorization occurs before successful authentication and refers to the process of obtaining approval to use a data set.|This must be one of "open", "registered", "controlled" or "private". "open" = openly accessible (default), "registered" = access will be granted to bona fide researchers only, "controled" = require to write a hypothesis, agree to terms, going to be vetted by a steering committee for access, "private" = closed dataset available only to the creator. Permission to access "registered" and "controlled" is givin by the creator.|

Reviewer: ```suggestion
|Privacy|privacy|**RECOMMENDED** The privacy level of the dataset.|This must be one of "open", "registered", "controlled" or "private". "open" = openly accessible (default); "registered" = access will be granted to bona fide researchers only; "controlled" = require to write a hypothesis, agree to terms, going to be vetted by a steering committee for access; "private" = closed dataset available only to the creator. Permission to access "registered" and "controlled" is given by the creator.|
```
